LICENSE.md

```diff
@@ -40,4 +40,4 @@ Map Data
 ========
 
 The default map data used comes from OpenStreetMap - <http://osm.org/copyright>
-The default imagery comes from MapQuest and is subject to the following terms: <http://developer.mapquest.com/web/info/terms-of-use> and <http://info.mapquest.com/terms-of-use/>
+The default imagery comes from MapQuest and is subject to the following terms: <https://developer.mapquest.com/legal> and <https://hello.mapquest.com/terms-of-use>
```

Comment:
Would be helpful to explain why there are _two_ links (not just one) and what they govern in detail ... in case somebody knows ...?

Reply:
hi @ptandler  guess I am still trying to figure it out, 
- I noticed one governs the developers ( https://developer.mapquest.com/legal ) and the other governs general terms of use (https://hello.mapquest.com/terms-of-use ) , its still not really that clear to me yet.